### DISCUSSION TITLE
How do I use Transmit client in Edge files?

### DISCUSSION THREAD

**Discussion by simonjcarr** [adonisjs/core#4575] [Help]
Is the Transmit client designed only to be used with Front end frameworks such as Vue or Rect etc?. Is there a way to use it with .edge files?, or ideally can I use HTMX SSE as a client?

**Comment by MaximeMRF** [comment]
You can use the transmit client with anything you want, personnaly i use it with a front in react, a android app with kotlin and a ios app with swift.
But for the android and ios app you have to understand the transmit protocol because there is no transmit client lib for theses languages

**Reply by simonjcarr** [reply]
I could not find any examples of integrating javascirpt into edge and mutating @foreach variables, so I have gone with HTMX in edge and returning  rendered components through SSE using resposne.response.send()
